src/server.js

```diff
@@ -21,5 +21,14 @@ app.use(
 )
 
 const server = Server(app)
-server.listen(PORT)
-logger.info(`Listening on ${PORT}`)
+server.listen(config.PORT)
```

Comment:
We can't begin accepting connections until the database is set up.

Reply:
Ya, my bad. Didn't think about it since it was on the server-side, not db-side. Will fix when I get a chance to sit down.